CoreGraphFramework/src/au/gov/asd/tac/constellation/graph/locking/LockingManager.java

```diff
@@ -145,12 +142,15 @@ public T startReading() {
     }
 
     protected void update(final Object description, Object editor) {
+        // Overridden in class DualGraph
```

Comment:
It looks like you deleted the overrides in `DualGraph`...

Reply:
Yeah I think I came across the issues in locking manager before the ones in DualGraph. I've gone back and removed the unused methods

Reply:
What I meant was, this comment is no longer valid as you've removed the overridden methods in DualGraph.

Reply:
The update method is still being overridden in DualGraph. This is the section where its used, specifically line 87
https://github.com/constellation-app/constellation/blob/98f4bc92108945a6c86ee22daad6c14518e0e120/CoreGraphFramework/src/au/gov/asd/tac/constellation/graph/locking/DualGraph.java#L84-L98